vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/jwt/claims/JwtTimeToLiveClaim.kt

```diff
@@ -0,0 +1,39 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList.jwt.claims
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.PositiveDuration
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.PositiveDurationSecondsJsonNumberSerializer
+import kotlinx.datetime.Instant
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmInline
+import kotlin.time.Duration
+
+/**
+ * source: https://www.rfc-editor.org/rfc/rfc7519
+ *
+ * OPTIONAL.
+ * The ttl (time to live) claim, if present, MUST specify the maximum amount of time,
+ * in seconds, that the Status List Token can be cached by a consumer before a fresh
+ * copy SHOULD be retrieved.
+ * The value of the claim MUST be a positive number encoded in JSON as a number.
+ */
+@Serializable
+@JvmInline
+value class JwtTimeToLiveClaim(
+    @Serializable(with = PositiveDurationSecondsJsonNumberSerializer::class)
+    val positiveDuration: PositiveDuration,
+) {
+    data object Specification {
+        const val CLAIM_NAME = "ttl"
+    }
+
+    fun isInvalid(
```

Comment:
This method is never called

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b

Well, at least a test has been provided